.gitignore

```diff
@@ -132,3 +132,4 @@ tmp_cache
 
 # python-language-server / Rope
 .ropeproject
+/env/Scripts
```

Comment:
Seems like you've added a line to the `.gitignore` here. It should just be the original, so this line shouldn't be here.
```suggestion
```